### DISCUSSION TITLE
Genesis omit Multitap support pin question

### DISCUSSION THREAD

**Discussion by pmgducati** [darthcloud/BlueRetro#247] [Q&A] [answered]
Hi @darthcloud ,
I'm working on the Genesis variant and in order for it to fit on the footprint of my PCBs that I have been working on I need to omit the multitap support my question is that if I omit U2 and U4, where does pin 4 on U3B and pin 10 on U3C need to go?  I saw in the notes that IO32 is GND with no Multitap so Pin 1 on U3A is good. 

![image](https://user-images.githubusercontent.com/20894227/162026244-fc8f658e-f25d-458b-8a5b-1a5f5d0a199e.png)

Thanks!

**Comment by darthcloud** [comment, marked as answer]
No love for multitap 😥😉

Plug them to GND and btw this configuration was tested by Dubesinhower too  <blockquote class="twitter-tweet"><p lang="en" dir="ltr"><a href="https://twitter.com/darthcloud64?ref_src=twsrc%5Etfw">@darthcloud64</a> I&#39;m trying to understand your Genesis schematic if I want to wire both controllers but not use multitap. Does this look right? <a href="https://t.co/IHalPTQLbG">pic.twitter.com/IHalPTQLbG</a></p>&mdash; Dubesinhower (@dubesinhower) <a href="https://twitter.com/dubesinhower/status/1466576602158956544?ref_src=twsrc%5Etfw">December 3, 2021</a></blockquote>

**Reply by pmgducati** [reply]
Awesome! Thanks!

**Comment by pmgducati** [comment]
@darthcloud 
One more quick question, should IO1 be left floating on the DB25 or tie it to GND also?

**Reply by darthcloud** [reply]
leave floating

**Reply by darthcloud** [reply]
Important one is IO32 to GND as it is a detection pins.

**Comment by pmgducati** [comment]
@darthcloud Finally getting around to making a genesis PCB (No Multitap) for myself and am having an issue.  For some reason controller one is always pressing right.  I have been looking at my schematic/PCB for two days now and cannot find an issue.  I'm using the same ESP32 for all systems and it works as intended on all the others (13 total). I have a Analogue Mega Sg (not sure if that is the issue) and running the current FW (1.8.1).

![image](https://user-images.githubusercontent.com/20894227/212414115-4a916505-d14c-492a-8722-f966edfd41f8.png)
![image](https://user-images.githubusercontent.com/20894227/212414158-262d4239-b610-44b9-bf52-56bf27e2f980.png)

Any thoughts would be appreciated!

Also for the record the Multitap version works great.

**Reply by darthcloud** [reply]
No idea it look fine, does a pullup on either IO23 or the Right line change anything?

**Reply by pmgducati** [reply]
No unfortunately that did not help, I did some further digging and found that when either P1 or P2 are plugged in alone they work as intended.  I tried Hooking up each wire for P2 and testing the connection and TH2 seems to be causing the issue.

**Reply by darthcloud** [reply]
You are using HW1 firmware right? or try on v1.7.3 to be sure the new internal stuff isn't interfering.
